### PR TITLE
ONEUP-6419 notification action

### PR DESCRIPTION
**Pull request by querwurzel**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                  "type": "mrkdwn",
+                  "text": "${{ toJSON(env.SUCCESS_MESSAGE) }}"
+                }
+              }]
+            }]
+          }
+
+    - name: Report - failure [on master]
+      if: failure() && github.ref == 'refs/heads/master'
+      uses: slackapi/slack-github-action@v1
```
Comment (yolch-yolchyan) [inline action.yml:67]: What would happen if we use v1.24.0?
https://github.com/slackapi/slack-github-action/tags

**Review by yolch-yolchyan — APPROVED** [review]

File: action.yml (hunk 1 above)
Comment (querwurzel) [reply action.yml:67]: v1 includes all v1.x.y.z releases .. you can see v1.24.0 and v1 pointing to the same commit (id)

File: action.yml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+runs:
+  using: composite
+  steps:
+    - if: always() && github.ref == 'refs/heads/master'
+      shell: bash
+      run: |
+        GIT_COMMIT_MSG=$(git log -1 --pretty=format:'%s')
+        GIT_COMMIT_MSG_NO_SUFFIX="${GIT_COMMIT_MSG#*\(#}"
+        GIT_PR="${GIT_COMMIT_MSG_NO_SUFFIX%\)*}"
```
Comment (catalin-cretu) [inline action.yml]: Can we make this slightly more complicated so we can handle [1up-bff commits](https://github.com/spring-media/1up-bff/blob/c0f9c1d693fc1b0d1a388e3d23edaeb89f77ba03/.github/workflows/bff-api.yml#L178) as well?

**Review by catalin-cretu — APPROVED** [review]

File: action.yml (hunk 2 above)
Comment (querwurzel) [reply action.yml]: I've tried to merge with bff .. could you check again?

**Pull request merged**